.github/workflows/generate-android-studio-releases.yml

```diff
@@ -0,0 +1,18 @@
+name: Generate Android Studio releases
+
+on:
+  schedule:
+    - cron: "1 1 * * *"
+
+jobs:
+  example-action:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Generate
+        run: .github/workflows/generate-android-studio-releases.sh
+      - name: Commit changes
+        uses: Endbug/add-and-commit@v8
+        with:
+          message: 'Generate Android Studio releases'
```

Comment:
include timestamp/date in message?

Reply:
There's no need for that – commit will have one.